src/uuid.erl

```diff
@@ -30,6 +30,7 @@
 %
 -module(uuid).
 -export([v4/0, to_string/1, get_parts/1, to_binary/1]).
+-ignore_xref([{v4, 0}, {to_string, 1}, {get_parts, 1}, {to_binary, 1}]).
```

Comment:
Same syntax as in "-export" can be used, btw (e.g. "-ignore_xref([v4/0, to_string/1, ...])."

Reply:
👍